test/udfs/test_utils.py

```diff
@@ -0,0 +1,28 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import cv2
+import numpy as np
+
+
+def get_udf_data_path():
+    return os.path.join(os.path.dirname(os.path.abspath(__file__)), "data")
```

Comment:
Is this path guaranteed to remain unchanged or is it part of the configuration file?

Reply:
Should we store all data at the same location -- like mnist.mp4 and image files?

Reply:
In my opinion, yes.
However it should be a separate PR, which I would be more than happy to create.
Since this PR was about Gaussian blurs I did not want to unnecessarily change unrelated parts of the code base.
I instead pulled this function out because I was using the exact same logic in my own tests.